pkg/kubestr/storage_provisioners_test.go

```diff
@@ -325,3 +325,28 @@ func (s *ProvisionerTestSuite) TestGetDriverNameFromUVSC(c *C) {
 		c.Assert(out, Equals, tc.out)
 	}
 }
+
+// func (s *ProvisionerTestSuite) TestGetDriverStats(c *C) {
```

Comment:
Why did you have to disable this ? Create a ticket to track this ?

Reply:
This test was just for some stat discovery. Not meant to be run all the time, if you see at the bottom I have an assert that fails everytime. It was just the quickest way for me to get some numbers.

Reply:
sounds good.